gradle/dependency-locks/testRuntimeClasspath.lockfile

```diff
@@ -12,20 +12,27 @@ com.google.guava:guava:19.0
 commons-codec:commons-codec:1.13
 commons-io:commons-io:2.6
 commons-lang:commons-lang:2.6
-junit:junit:4.12
-net.bytebuddy:byte-buddy-agent:1.10.15
-net.bytebuddy:byte-buddy:1.10.15
+net.bytebuddy:byte-buddy-agent:1.11.1
```

Comment:
where this is used?

Reply:
` ./gradlew dependencyInsight --dependency net.bytebuddy:byte-buddy-agent --configuration testRun
timeClasspath`:

```
net.bytebuddy:byte-buddy-agent:1.11.1
\--- org.mockito:mockito-core:3.11.0
     \--- testRuntimeClasspath
```